iam/src/index.ts

```diff
@@ -590,8 +594,16 @@ app.post("/api/v0.0.0/eas/passport", (req: Request, res: Response): void => {
         );
 
         const fee = await getEASFeeAmount(2);
+
         const passportAttestation: PassportAttestation = {
-          multiAttestationRequest,
+          multiAttestationRequest: multiAttestationRequest.map((request) => ({
```

Comment:
Does it make more sense to change the return from formatMultiAttestationRequestWithPassportAndScore? Or is that used elsewhere?

Reply:
Good catch. This actually needed removed!